corehq/apps/locations/bulk_management.py

```diff
@@ -168,8 +169,19 @@ def from_excel_row(cls, row, index, location_type):
         longitude = row.get(cls.titles['longitude'])
```

Comment:
Didn't actually make lat/lng as optional since we expect users to just use the downloaded excel file which already contains these